Btu-building-philadelphia/index.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+
+<html>
+    <head>
+        <title>Energy Consumption of Buildings in Philadelphia</title>
+        <meta name="viewport" content="width=device-width, initial-scale=1" />
+
+        <link rel="stylesheet" href="https://unpkg.com/leaflet@1.9.4/dist/leaflet.css"
+        integrity="sha256-p4NxAoJBhIIN+hmNHrzRCf9tD/miZyoHS5obTRR9BMY="
+        crossorigin=""/>
+        
+        <link rel="stylesheet" href="css/style.css">
+
+
+    </head>
+
+    <body>
+        <header> 
+            <h1>Building Energy Usages in Philadelphia (2021) </h1>
+        </header>
+
+        <main>
+            <div calss="left-side">
+
+                <section class="address-section">
+                    <input type="text" name="address-entry" id="address-entry" placeholder="Enter your building address here..." size="55">
+                    <ol id="address-choices" class="address-choices"></ol>
+                </section>
+
+                <section id="building-filters">
+                    <label><input type="checkbox" name="building-filter-type" value="Residential"> Residential</label>
+                    <label><input type="checkbox" name="building-filter-type" value="Commercial"> Commercial</label>
+                    <label><input type="checkbox" name="building-filter-type" value="Educational"> Educational</label>
+                    <label><input type="checkbox" name="building-filter-type" value="Hospital"> Hospital</label>
+                    <label><input type="checkbox" name="building-filter-type" value="Office"> Office</label>
+                    <label><input type="checkbox" name="building-filter-type" value="Cultural"> Cultural</label>
+                    <label><input type="checkbox" name="building-filter-type" value="Other"> Other</label>
+                    
+
+
+                </section>
+
+                <section class="building-list-section">
```

Comment:
**accessibility suggestion:** The Axe developer tools raised this as an issue. Since you have `overflow-y: auto` in the styles for this element, it may have a scroll bar when there are more list items than fit on screen. That's fine, as long as the user has a mouse to use. If they're navigating by keyboard however, they'll never be able to get to the element in order to scroll it. To fix this you could say:
```suggestion
                <section class="building-list-section" tabindex="0">
```
You can see more information about this issue at https://dequeuniversity.com/rules/axe/4.8/scrollable-region-focusable